src/daemon/keymap.c

```diff
@@ -939,6 +939,26 @@ const unsigned char harpoon_wl_lut[BRAGI_ONE_BYTE_MOUSE_BUTTONS] = {
     0x08, // but we need them to be non zero so that they don't undo left click
 };
 
+// DPI cycle/up and forwards are just swapped compared to corsair_bragi_lut
+const unsigned char m55_wl_lut[BRAGI_ONE_BYTE_MOUSE_BUTTONS] = {
+    0x00,
+    0x01,
+    0x02,
+    0x04,
+    0x03,
+    0x06, //forwards
+    0x05, //backwards
+    0x08, //dpi up?
+    0x09,
```

Comment:
BRAGI_ONE_BYTE_MOUSE_BUTTONS is 8 but there are more than 8 entries here. The compiler should be throwing a warning.

Reply:
Shortened LUT to 8 entries